root.tf

```diff
@@ -250,47 +250,64 @@ module "dirty_upload_sns_topic" {
   sns_policy  = "s3_upload"
 }
 
+module "backend_check_sqs_queue" {
```

Comment:
Maybe rename the module: "backend_check_failure_sqs_queue"?

Reply:
Yeah, good point. I've changed it.